tests/resource_tests/test_kafka.py

```diff
@@ -46,16 +45,8 @@ def test_kafka_inject_success(kafka_resource, passing_injections, test_df, mocke
     kafka_resource.write(test_df)
     mocked_kafka_producer.send.assert_has_calls(
         [
-            call(passing_injections["var1"], key=0, value={"a": 1, "b": 4}),
-            call(passing_injections["var1"], key=1, value={"a": 2, "b": 5}),
-            call(passing_injections["var1"], key=2, value={"a": 3, "b": 6}),
+            call(passing_injections["var1"], key=0, value={"a": 1, "b": "x", "c": True}),
+            call(passing_injections["var1"], key=1, value={"a": 2, "b": "y", "c": False}),
+            call(passing_injections["var1"], key=2, value={"a": 3, "b": "z", "c": True}),
         ]
     )
-
-
-def test_kafka_inject_fail(kafka_resource, failing_injections, test_df, mocked_kafka_producer):
```

Comment:
Any reason why we don't want this test anymore?

Reply:
Injections don't fail any more - at least not in that way. They now only fail when trying to inject an attribute that is not injectable (not a string path or none). There is still the `check_injections` function in `inject.py` but it is currently unused. I could put it back in.